src/Files.App/Views/Properties/GeneralPage.xaml

```diff
@@ -36,19 +36,16 @@
 	</vm:BasePropertiesPage.Resources>
 
 	<ScrollViewer>
-		<StackPanel
-			Padding="12"
-			Spacing="4"
-			Style="{StaticResource PropertiesTab}">
+		<StackPanel Padding="12" Spacing="8">
 
-			<!--  Item overview  -->
+			<!--  Item Overview  -->
 			<Grid
 				Padding="12"
 				Background="{ThemeResource CardBackgroundFillColorDefaultBrush}"
 				BorderBrush="{ThemeResource CardStrokeColorDefaultBrush}"
 				BorderThickness="1"
 				ColumnSpacing="8"
-				CornerRadius="4"
+				CornerRadius="8"
```

Comment:
Should there be a resource for the corner radius in app.xaml? This will allow us to make future changes without having to update every instance.

Reply:
No, might have to create 'CardCornerRadius' like 'OverlayCornerRadius', 'ControlCornerRadius'.

Reply:
I meant that we should create one, in the meantime we can keep this change but let's create a resource in a separate PR.